Documentation/General/finalReport/appendix-neuralnetwork.tex

```diff
@@ -0,0 +1,41 @@
+\chapter{Neural Networks}
+
+One of the bigger issues we had to deal with in MoodCat was to classify songs
+to moods. The initial plan was to create a neural network that could do this.
+
+Some of the MoodCat team members have followed a course about computational intelligence,
+in which different ways to use learning algorithms to solve different kinds of problems.
+One widely used technique to classify objects by using the objects' features as inputs
+is the use of neural networks.
+
+Neural networks consist of an input layer, an output layer and a number of hidden layers, 
+and each layer consists of a number of nodes.
+
+The nodes in the input layer are fed with 'features', if we take classifying songs as example
+this would be all kinds of information of the songs that can be represented as some form
+of numeric metrics. The amount of BPM of a song could be a metric, or the overall loudness
+of songs. The key of a song is not a good metric because it cannot be represented as a scale,
+e.g. music in A is not more like music in B than like music in G, while in numeric form A would be closer to B and on the other end of G.
```

Comment:
cannot be represented on a linear scale, for example music in A has not more in common with B than C or even G.
While on a linear scale A would deviantly have more in common with B than G.